Sources/SkyFloatingLabelTextField.swift

```diff
@@ -74,21 +74,23 @@ open class SkyFloatingLabelTextField: UITextField { // swiftlint:disable:this ty
     }
 
     fileprivate func updatePlaceholder() {
-        if let placeholder = placeholder, let font = placeholderFont ?? font {
-            #if swift(>=4.0)
-                attributedPlaceholder = NSAttributedString(
-                    string: placeholder,
-                    attributes: [
-                        NSAttributedStringKey.foregroundColor: placeholderColor, NSAttributedStringKey.font: font
-                    ]
-                )
-            #else
-                attributedPlaceholder = NSAttributedString(
-                    string: placeholder,
-                    attributes: [NSForegroundColorAttributeName: placeholderColor, NSFontAttributeName: font]
-                )
-            #endif
+        guard let placeholder = placeholder, let font = placeholderFont ?? font else {
+            return
         }
+        let color = isEnabled ? placeholderColor : disabledColor
+        #if swift(>=4.0)
+            attributedPlaceholder = NSAttributedString(
+            string: placeholder,
+            attributes: [
+            NSAttributedStringKey.foregroundColor: color, NSAttributedStringKey.font: font
```

Comment:
Please cleanup this indentation to match what it was before.

Reply:
Fixed!